cmd/client/main.go

```diff
@@ -344,13 +344,13 @@ func (c *Client) getUDSDialer(o *GrpcProxyClientOptions) (func(ctx context.Conte
 			}
 			return c, err
 		})
-		tunnel, err := client.CreateSingleUseGrpcTunnel(o.proxyUdsName, dialOption, grpc.WithInsecure(), grpc.WithUserAgent(o.userAgent))
+		tunnel, err := client.CreateSingleUseGrpcTunnel(context.Background(), o.proxyUdsName, dialOption, grpc.WithInsecure(), grpc.WithUserAgent(o.userAgent))
```

Comment:
Not worth holding the PR for but it would be a better example if we used 
```ctx := context.Background()```
then passed ctx in on these calls.

Reply:
Done.